test/e2e/es/license_test.go

```diff
@@ -46,7 +46,7 @@ func TestEnterpriseLicenseSingle(t *testing.T) {
 			licenseTestContext.CreateEnterpriseLicenseSecret(licenseBytes),
 		}).
 		// Mutation shortcuts the license provisioning check...
-		WithSteps(mutatedEsBuilder.MutationTestSteps(k)).
+		WithSteps(mutatedEsBuilder.MutationTestSteps(k, test.MutationOptions{})).
```

Comment:
You were explicit about `IncludesRollingUpgrade` in all cases except this one, any reason in particular?

Reply:
The "mutation" here is kind of a hack since no mutation actually happens. Picking either `true` or `false` for `IncludesRollingUpgrade ` is a hard choice 😄
(happens to end up `false` by default though 🤷‍♂ )